### PR TITLE
nitrpg: more statistics

### PR DESCRIPTION
**Pull request by Morriar**
Some changes on the stats API and new stats for Players.

### CODE REVIEW

File: contrib/nitrpg/src/statistics.nit [hunk 1]
```diff
@@ -59,44 +85,57 @@ end
...
 
+	redef var game
+
 	redef var key = "statistics"
 
 	# Used internally to stats values.
-	private var stats = new HashMap[String, Int]
-
-	init do clear
+	private var values = new HashMap[String, Int]
```
Comment (privat) [inline contrib/nitrpg/src/statistics.nit]: Counter pour profiter deja d'opérations statistiques avancées?

**Comment by privat** [comment]
+1

**Comment by Morriar** [comment]
Reroll'd

* [x] Stats inherits Counter as suggested by @privat

**Comment by Morriar** [comment]
@all ping for +1

**Comment by R4PaSs** [comment]
+1, c'est cool de voir comment on récupère les Nitcoins :)

**Comment by Morriar** [comment]
Even within a beta you can be ban for exploit :p

Alexandre

**Comment by R4PaSs** [comment]
Hehe, that was unexpected, but hey, 2 free Nitcoins ! :)

**Pull request merged**